src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -586,7 +588,7 @@ public Replacement(string tag, RoleName role, MultiNodeSpec spec)
 
         protected void InjectDeployments(ActorSystem system, RoleName role)
         {
-            var deployer = Sys.AsInstanceOf<ExtendedActorSystem>().Provider.Deployer;
+            var deployer = system.AsInstanceOf<ExtendedActorSystem>().Provider.Deployer;
```

Comment:
Nasty bug - needed to use the instance variable rather than the property `Sys` to work, otherwise we could never successfully call `MultiNodeSpec.NewActorSystem()` and get a correct result if we used any programmatic deployments.